examples/tutorial-standalone/main.tf

```diff
@@ -0,0 +1,198 @@
+/**
```

Comment:
are we missing a templated `tf.vars` file to use as an example to populate inputs for user?  I think we used to have one.

Reply:
We added the `terraform.example.tfvars` file